frontend/templates/troubleshooting/solution.tsx

```diff
@@ -206,9 +206,11 @@ export default function SolutionCard({
    index: number;
    solution: SolutionSection;
 }) {
+   const solutionHeadingId = solution.heading.toLowerCase().replace(/\s/g, '-');
```

Comment:
This also needs to use the new function I think.

Reply:
Good lookin' out 👍